### ISSUE TITLE
false positive: the second argument to the assert statement is not a condition value

### ISSUE THREAD

**Issue by afg984** [PyCQA/pylint#1331]
### Steps to reproduce
run pylint aginst the code:
```python
def func(*args):
    assert False, len(args)
```

### Current behavior
```
C:  2,18: Do not use `len(SEQUENCE)` as condition value (len-as-condition)
```

### Expected behavior
the second argument to the assert statement is not a condition value

### pylint --version output
```
No config file found, using default configuration
pylint 2.0.0,
astroid 1.5.0
Python 3.6.0 (default, Jan 16 2017, 12:12:55)
[GCC 6.3.1 20170109]
```

**Comment by rogalski** [comment]
Thanks for bug report! I'm able to reproduce an issue.

**Comment by rogalski** [comment]
Fixed by ea964a0
